settings/provisioner/virsh/virsh.spec

```diff
@@ -3,38 +3,38 @@ subparsers:
     virsh:
         help: Provision systems using virsh
         groups:
-            - title: host
+            - title: Hypervisor
               options:
-                  host:
-                      type: str
+                  host-address:
+                      type: Value
```

Comment:
Unable to override default values for the custopm clg type in the ini file provided with --from-file argument. For example: 
I have in ini file
topology-network=default.yml
topology-nodes=1_controller
host-address=rhos-qe-06.lab.eng.brq.redhat.com

But provisioner is still using default values. 

The same is mostly likely actual for environment variables.